src/cbrkit/sim/graphs/_alignment.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Callable, List, Tuple, Any
+from ._model import Graph, Node, is_sequential
+from ...helpers import dist2sim
+
+__all__ = []
+
+try:
+    import numpy as np
+    from ..collections import dtw
+
+    @dataclass(slots=True)
+    class GraphDTW:
```

Comment:
This should inherit from SimFunc and (possibly) be generic as well. You can also make it frozen. Lastly, it uses a different name than is added to `__all__`. Please rename it to `dtw` as it is already in the subpackage `cbrkit.sim.graphs`